elyra/tests/pipeline/resources/sample_pipelines/pipeline_valid_with_pipeline_default.json

```diff
@@ -99,7 +100,8 @@
             ],
             "mounted_volumes": [
               { "path": "/mnt/vol2", "pvc_name": "pvc-claim-2" }
-            ]
+            ],
+            "disable_node_caching": "True"
```

Comment:
If the front end can persist boolean types (per the previous question), it might be good to have one value that is derived from a boolean.